platypus/parse.py

```diff
@@ -130,7 +124,8 @@ def parse_second_database(db, best_hits, percentage_ids_other,
 
 def process_results(percentage_ids, alignment_lengths, percentage_ids_other,
                     alignment_lengths_other, best_hits):
-    """Writes the individual rarefaction levels and then returns the collated result
+    """Writes the individual rarefaction levels and then returns the collated
```

Comment:
Not sure what you mean. However, the plan is to fix the docstring in a separate pr.